source/assets/critical/_header.scss

```diff
@@ -229,61 +232,76 @@
 /* THEME-SPECIFIC SETTINGS */
 .masthead.masthead--theme-sport {
   --theme-color-gradient-1: #{$theme-sport-color-1};
+  --theme-color-loader: #{$color-srf-red-600};
```

Comment:
hm, hat das was mit den Audios/Videos zu tun?

Reply:
Oha – nix, ist leider in diesen Branch gerutscht 🤔 drin lassen oder rauslöschen? 🤔

Reply:
lass es, solange es nicht woanders vermisst wird oder lustige Auswirkungen hat :)